### DISCUSSION TITLE
What's coming after USWDS 3.0?

### DISCUSSION THREAD

**Discussion by maya** [uswds/uswds#4383] [General]
Do you have any idea of what themes you’ll be focusing on after releasing [USWDS 3.0](https://github.com/uswds/uswds/discussions/4355) early next year?

**Comment by MsWebGirl** [comment]
You asked and so you shall receive!  ;-)

- A single official drupal distro instead of all the multiple versions out there. 

- Scrollable and responsive tables for agencies that use them 

- Guidance for visualizing data in an accessible, meaningful way

- Guidance on setting up accessibility testing in our dev environments to ensure 508 compliance
